[zpreztorc] Use an array to load the plugins for ease of use

samcv opened the pull request:
Previously we used a newline separated list of arguments which had
the downside of requiring a backslash at the end of each newline,
and in many cases made it difficult to comment out lines to disable
particular plugins. By using an array we can easily comment them out
and not have to deal with using backslashes at the end of lines.

Note: it may be a good idea to list all of the plugins, but leave some
commented out, making it even easier for people to enable plugins?
Even if not, this would make it much easier to do things like store
the array in another file if we decide to changeup the setup of 
zpreztorc and have a zpreztorc.local or something.

indrajitr commented:
> Note: it may be a good idea to list all of the plugins, but leave some
commented out, making it even easier for people to enable plugins?

Yes, having them listed (but commented out) also makes it easier for people to figure out the relative ordering of the respective plugins, should they chose to enable.

samcv commented:
I've added at least the modules I use commented out. I have also converted all the other settings which
used to take multiple arguments split across lines with backslashes to continue onto the next line.

belak commented:
I like the general idea behind this change but I think it might be best to put this on hold while we figure out what we're doing with runcoms (including zpreztorc)

samcv commented:
I agree, we can hold off until we figure out what we are doing with the runcoms.

sorin-ionescu commented:
The reason why a zstyle was used is to keep the settings format consistent.
